04-prompt-engineering-fundamentals/aoai-assignment.ipynb

```diff
@@ -72,31 +93,44 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
+   "execution_count": 24,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-06-03T11:25:11.310749Z",
+     "start_time": "2024-06-03T11:25:10.866201Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "By the dawn's early light\n"
+     ]
+    }
+   ],
    "source": [
     "# The OpenAI SDK was updated on Nov 8, 2023 with new guidance for migration\n",
     "# See: https://github.com/openai/openai-python/discussions/742\n",
     "\n",
     "## Updated\n",
     "import os\n",
-    "from openai import AzureOpenAI\n",
     "from dotenv import load_dotenv\n",
     "load_dotenv()\n",
     "\n",
-    "client = AzureOpenAI(\n",
-    "  api_key=os.environ['AZURE_OPENAI_KEY'],  # this is also the default, it can be omitted\n",
-    "  api_version = \"2023-05-15\"\n",
-    "  )\n",
+    "from openai import OpenAI\n",
+    "\n",
+    "client = OpenAI(\n",
+    "  api_key=os.environ['OPENAI_API_KEY'],  # this is also the default, it can be omitted\n",
+    ")\n",
     "\n",
-    "deployment=os.environ['AZURE_OPENAI_DEPLOYMENT']\n",
+    "deployment=os.environ['MODEL']\n",
     "\n",
     "## Updated\n",
     "def get_completion(prompt):\n",
     "    messages = [{\"role\": \"user\", \"content\": prompt}]       \n",
     "    response = client.chat.completions.create(   \n",
-    "        model=deployment,                                         \n",
+    "       model=\"gpt-3.5-turbo\",                                      \n",
```

Comment:
I would suggest using `deployment` for setting the model everywhere (as you do further) to be more consistent